contracts/contract_vms.md

```diff
@@ -21,14 +19,18 @@ The current meaning of this field is:
 |   00        |          |             | NO_VM (Used in oracles)
 |   01        | [AEVM_01](aevm.md)  | [Sophia_01](sophia.md#the-sophia_01-abi)   | For Sophia contracts on the AEVM
 |   02        | [AEVM_01](aevm.md)  | [Solidity_01](solidity.md#the-solidity_01-abi) | For Solidity contracts on the AEVM
-|   03        | [FTWVM_01](contract_vms.md#ftwvm) | Sophia_02   | For Sophia contracts on the FTWVM
-|   04        | [HLM_01](contract_vms.md#hlm)   | Varna_01    | For Varna contracts on the HLM
-|   05        | [FAEVM_01](aevm.md#the-fast-%C3%86ternity-virtual-machine-faevm) | [Solidity_01](solidity.md#the-solidity_01-abi)| For fast execution of Solidity contracts
-| 06-FF       |          |             | UNUSED
+|   03        | [AEVM_02](aevm.md)  | [Sophia_01](sophia.md#the-sophia_01-abi)   | Containing improvments of the AEVM for Sophia
+|   04-FF     |          |             | UNUSED
+
+Which VM versions are accepted are different based on consensus protocol versions.
 
-*NOTE:* Currently the only value accepted in ContractCreateTx, ContractCallTx, and ChannelForceProgressTx is `0x01`.
+| Protocol version | Operation           | Accepted VM values |
+| ---------------- | ---------           | ------------------ |
+| Roma             | all                 | `0x01`             |
+| Minerva          | contract call       | `0x01`, `0x03`     |
```

Comment:
So far the aevm versions is compatible for calls since the ABI has not changed. In other words, all contracts will run under the VM they are created for, but you can call between contracts of different vm versions.

In the code, we only allow calls AEVM_02->AEVM_01, but as long as the ABI remains stable we can probably lift this restriction.

Force progress is a contract call in this respect, and follows the same restrictions.

Testing this is tracked by a PivotalTracker [ticket](https://www.pivotaltracker.com/story/show/162578883)

Reply:
> but you can call between contracts of different vm versions.

I assume this includes different VM versions in call tx and contract. Ok.